projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -34,12 +34,9 @@ public static function on_jetpack_feature_publicize_enabled() {
 
 		if ( ! isset( $publicize_ui ) ) {
 			$publicize_ui = new Publicize_UI();
-
 		}
 
-		// Adding on a higher priority to make sure we're the first field registered.
-		// The priority parameter can be removed once we deprecate WPCOM_REST_API_V2_Post_Publicize_Connections_Field
-		add_action( 'rest_api_init', array( new Connections_Post_Field(), 'register_fields' ), 5 );
+		new Connections_Post_Field();
```

Comment:
Will this be initialized on Simple sites?

Reply:
I don't think the `on_jetpack_feature_publicize_enabled` method here is called on WPCOM. We need an unconditional initialization for WPCOM, the way [we do for assets](https://github.com/Automattic/jetpack/blob/9f6b92002afa3d3812b0018923adb2d3f0a494a6/projects/packages/publicize/actions.php).

Probably, we should have a method named `unconditional_initialization` inside `Publicize_Setup` class to add all that logic there and also more that assets initialization logic into that.

Reply:
Yes we would need to adjust the loading of it on WPCOM too